src/library_webgl2.js

```diff
@@ -622,23 +622,23 @@ var LibraryWebGL2 = {
     assert((value & 3) == 0, 'Pointer to integer data passed to glClearBufferiv must be aligned to four bytes!');
 #endif
 
-    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, value>>2);
+    GLctx.clearBufferiv(buffer, drawbuffer, HEAP32, {{{ getHeapOffset('value', 'float') }}});
```

Comment:
This and some others later are not floats (but they are 4 bytes, so it happens to work).